### DISCUSSION TITLE
primeblocks-3.2.0 isn’t connected to my PrimeOne-2.2.0 library.

### DISCUSSION THREAD

**Discussion by dsodamini** [primefaces/community#947] [Figma UI Kit]
Hello!

The problem is that primeblocks-3.2.0 isn’t connected to my PrimeOne-2.2.0 library.
So I tried to link them in the following way(my PrimeOne-2.2.0 has been selected from the "Choose library" dropdown):
<img width="515" alt="image" src="https://github.com/primefaces/community/assets/156133480/a9ddac3f-fcac-46ff-97c9-fd11af062723">
As a result some of the components(mostly buttons and inputs) got broken. Can you suggest how to connect these libraries correctly?
Here are some examples(there are many more in my file):
Broken(linked to the main library):
<img width="745" alt="image" src="https://github.com/primefaces/community/assets/156133480/a5c83262-414f-438b-a82e-3733c5ae9133">
<img width="895" alt="image" src="https://github.com/primefaces/community/assets/156133480/428e76b2-6e85-47ec-8a00-fd606f5eca15">
<img width="794" alt="image" src="https://github.com/primefaces/community/assets/156133480/42edc368-4e9c-4ce4-89f4-e6842ec782e8">
vs Normal(not linked to the main library):
<img width="829" alt="image" src="https://github.com/primefaces/community/assets/156133480/59c59c6a-7edf-419d-bdd2-68b5cb6723b4">
<img width="848" alt="image" src="https://github.com/primefaces/community/assets/156133480/df93353a-642c-4f0b-8fbd-e2136599e9f1">
<img width="730" alt="image" src="https://github.com/primefaces/community/assets/156133480/b4c08aff-320d-4692-b52a-1f3a74597ec5">

Can you please help?

**Comment by onursenture** [comment]
Yes, the current issue is because the versions of PrimeOne 2.2.0 and PrimeBlocks you have were not released simultaneously, so the PrimeOne version in PrimeBlocks is outdated.

We're working on the next version of PrimeBlocks, which will be released with a Figma file compatible with PrimeOne 2.2.0.

Unfortunately, there's no easy and quick way around this process, so we must make all the modifications. I appreciate your understanding.

**Reply by dbsmalltweaks** [reply]
What version of PrimeOne is compatible with PrimeBlock 3.2.0?

**Reply by onursenture** [reply]
@dbsmalltweaks 1.0.0

**Comment by dsodamini** [comment]
Thanks for the info! 

When can we expect the next version? 

Best, 
Dino

**Reply by onursenture** [reply]
Unfortunately, we cannot estimate right now because the next PrimeBlocks version will not only be a version in which new blocks are added but also a significant release in which all blocks will be implemented with Tailwind, and our work on this continues.

**Reply by ren-x7** [reply]
@onursenture will it be this year tho?

**Reply by onursenture** [reply]
Yes.
